packages/schematics/update/update/npm.ts

```diff
@@ -133,7 +133,14 @@ export function getNpmPackageJson(
     },
   );
 
-  const response = from<NpmRepositoryPackageJson>(resultPromise).pipe(shareReplay());
+  const response = from<NpmRepositoryPackageJson>(resultPromise).pipe(
+    shareReplay(),
+    catchError(err => {
+      logger.warn(err);
+
+      return EMPTY;
```

Comment:
use `empty()` instead of `EMPTY`.

Reply:
Disregard, `EMPTY` is fine (see https://rxjs.dev/api/index/const/EMPTY)